.travis.yml

```diff
@@ -6,9 +6,9 @@ script: bash -ex .travis-opam.sh
 sudo: true
 env:
   matrix:
-  - PACKAGE="decompress" OCAML_VERSION=4.03 TESTS=true PINS="optint:https://github.com/dinosaure/optint.git checkseum:https://github.com/dinosaure/checkseum.git" 
-  - PACKAGE="decompress" OCAML_VERSION=4.04 TESTS=true PINS="optint:https://github.com/dinosaure/optint.git checkseum:https://github.com/dinosaure/checkseum.git" 
-  - PACKAGE="decompress" OCAML_VERSION=4.05 TESTS=true PINS="optint:https://github.com/dinosaure/optint.git checkseum:https://github.com/dinosaure/checkseum.git" 
-  - PACKAGE="decompress" OCAML_VERSION=4.06 TESTS=true PINS="optint:https://github.com/dinosaure/optint.git checkseum:https://github.com/dinosaure/checkseum.git" 
+  - PACKAGE="decompress" OCAML_VERSION=4.03 TESTS=true PINS="optint:https://github.com/dinosaure/optint.git checkseum:https://github.com/dinosaure/checkseum.git#crc32"
```

Comment:
`#crc32` is now merged, should remove this branch pin before merging.

see https://github.com/mirage/checkseum/pull/14